source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
+
+async function disableWikiAndProjects(): Promise<void | false> {
+	if (sessionStorage.rghNewRepo !== getRepoURL()) {
+		return false;
+	}
+
+	sessionStorage.removeItem('rghNewRepo');
+
+	api.v3(`repos/${getRepoURL()}`, {
+		method: 'PATCH',
+		body: {
+			has_projects: false,
+			has_wiki: false
+		}
+	});
+
+	const wiki = await elementReady('[data-tab-item="wiki-tab"]');
+	wiki?.closest('.d-flex')!.remove();
+	const projects = await elementReady('[data-content="Projects"]');
+	projects?.closest('.d-flex')!.remove();
+}
+
+function setStorage(): void | false {
+	if (!select<HTMLInputElement>('[name="rgh-disable-project"]')!.checked) {
+		return false;
+	}
+
+	const owner = select('#repository-owner')!.textContent!.trim();
+	const repository = select<HTMLInputElement>('.js-repo-name')!.value;
+	sessionStorage.rghNewRepo = owner + '/' + repository;
```

Comment:
`sessionStorage` seems to only be set for the current tab, so as long as:

1. we set it on submit
2. we remove it on the next page load

It's nearly impossible that the page load is not the same repo we just created.


```suggestion
	sessionStorage.rghNewRepo = true;
```